### PR TITLE
release notes for 1.16.2

### PR DESCRIPTION
**Pull request by AtnNn**
@danielmewes and @mglukhovsky could you please review?

### CODE REVIEW

**Comment by danielmewes** [comment]
Looks good to me.

File: NOTES.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+# Release 1.16.2 (Stand By Me)
+
+Released on 2015-02-16
+
+Bug fix update.
+
+* Fixed a bug in `r.range` that caused a crash in debug mode (#3767)
```
Comment (Tryneus) [inline NOTES.md]: This would also cause queries to fail in release mode - so I would just say "that caused query failures"

**Comment by timmaxw** [comment]
"error" is misspelled.

File: NOTES.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+* Fixed a race condition in the implementation of `.order_by.limit.changes` (#3765)
+* Fixed a build errror that caused really slow `r.js` performance (#3757)
```
Comment (timmaxw) [inline NOTES.md]: This might be nitpicking, but I'd replace "really" with "very"; "really" sounds too informal to me.

**Comment by mglukhovsky** [comment]
Agreed with both @tim and @Tryneus' comments. Otherwise, looks good.

**Comment by AtnNn** [comment]
Thanks. Merged in 1427bd0.

**Pull request closed without merging**